charts/bandstand-web-service/templates/_deployment.yaml

```diff
@@ -95,12 +95,14 @@ spec:
               drop:
                 - all
       volumes:
+        { { - if or .Values.properties .Values.envProperties } }
         - name: config
           configMap:
```

Comment:
Would it not fail on line 42 for `volumeMount` ? if the volume `config` isn't defined ?

Reply:
Yes, that needs wrapping in the same if statement

Reply:
added  at 42